collection-scripts/gather

```diff
@@ -49,3 +49,12 @@ done
 
 # Gather Windows Kubernetes component logs
 /usr/bin/gather_windows_node_logs
+
+# Gather OVN traces 
+/usr/bin/gather_network_ovn_trace
```

Comment:
just to confirm @astoycos this wont cause any failures if it is run on openshift-sdn cluster right? It just wont collect anything?

Reply:
Right this (https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_network_logs#L131) Checks what type of networking is being used before gathering